ui/src/ui/pages/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Dashboard = () => {
+  return (
+    <div>
+    </div>
+  );
+};
+
+export default Dashboard;
```

Comment:
Why do you need this empty component?

Reply:
I was navigating to dashboard after signin/signup. That's why, just created it!